src/files-and-videos/videos-page/transcript-settings/TranscriptSettings.jsx

```diff
@@ -117,11 +117,13 @@ const TranscriptSettings = ({
         {(!transcriptType && isAiTranslationsEnabled) && (
```

Comment:
Can all of this go into the plugin? I think the plugin should be for additional translation options not just ai translations.

Reply:
The check should go into the plugin because they are specific to the ai translations. The value should be able to be fetched from inside the plugin as it is in the redux store.